apps/src/templates/Button.jsx

```diff
@@ -148,6 +149,12 @@ const styles = {
       paddingLeft: 30,
       paddingRight: 30,
       lineHeight: '40px'
+    },
+    [ButtonSize.small]: {
+      height: 40,
+      paddingLeft: 10,
+      paddingRight: 10,
+      lineHeight: '40px'
```

Comment:
I'm being extra finicky but maybe `narrow` instead of `small`? Looks like the default size is shorter, but this new one has less L/R padding.

Reply:
https://github.com/code-dot-org/code-dot-org/pull/32444/commits/8fa50515035d5b1760e9eef95757b48e785efcf3